system-int-test/test/cmr/system_int_test/search/variable/variable_association_test.clj

```diff
@@ -85,18 +85,17 @@
             _ (index/wait-until-indexed)
             response (association-util/associate-by-concept-ids
                       token concept-id [{:concept-id c1-p1}])]
-        (vu/assert-variable-association-bad-request
-         {[c1-p1] {:errors [(format "Collection [%s] does not exist or is not visible." c1-p1)]}}
-         response)))
-
+        ;; when the collection was deleted, the variable got deleted too
```

Comment:
Please update variable_ingest_with_association_test.clj to make the collection delete test stands out with a `testing` clause rather than a comment (that is why I missed it). Then delete the testing here as it is a duplication.

Reply:
Done